### ISSUE TITLE
RC4 is rather broken

### ISSUE THREAD

**Issue by FredyC** [stampit-org/stampit#216]
First issue is with calling `create(options)` ... I've addressed and fixed this in #215

Another issue comes from removal of `es2015-rollup` preset. Apparently these `external-helpers` are quiet necessary as without that resulting bundle is missing this critical line:
```js
Object.defineProperty(exports, '__esModule', { value: true });
```
When you try import module without this line in Babel enabled environment, you can see code like this in there.
```js
var _stampit = require('stampit');
var _stampit2 = _interopRequireDefault(_stampit);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
As it cannot find `__esModule` in there, it wont be able to recognize ES6 and suddenly you have `_stampit2.default.default.methods` which obviously doesn't.

I don't think that `external-helpers` are that big in the end to be worth the hassle, so I suggest reverting these changes completely.

**Comment by koresar** [comment]
I clearly see
`  Object.defineProperty(exports, '__esModule', { value: true });`
in `dist/stampit.js`, `dist/stampit.full.js`, and `dist/stampit.es5.js`. It's not present in the `.mjs` file, which is correct.

Node v0.10
```
┌vasyl@192-168-1-52 ~/c/stampit (v3_0 $)                              
└> nvm use 0
Now using node v0.10.40 (npm v2.15.6)
┌vasyl@192-168-1-52 ~/c/stampit (v3_0 $)                              
└> node
> var s=require('./');
undefined
> typeof s.default
'function'
> typeof s
'object'
> s.__esModule
true
```

And node v6.
```
┌vasyl@192-168-1-52 ~/c/stampit (v3_0 $)
└> nvm use 6
Now using node v6.2.0 (npm v3.8.9)
┌vasyl@192-168-1-52 ~/c/stampit (v3_0 $)                              
└> node
> var s=require('./');
undefined
> typeof s.default
'function'
> typeof s
'object'
> s.__esModule
true
```

**Comment by koresar** [comment]
Side note. I'm starting to be more convinced of ditching the ES6 from the stampit source code. This is getting ridiculous. :(

**Comment by koresar** [comment]
So, I've just pushed few more commits.
* Fixed the coverage I broke. It's weird, but the only way to fix it was the `.babelrc` file I removed earlier.
* As the result we don't need to pass babel presets anywhere.
* Also I upgraded every devDependency to the latest possible.
* Travis [is green](https://travis-ci.org/stampit-org/stampit/builds/137393008).

Now, it's time to think of the browser tests. Any ideas?
